### PR TITLE
Feature/update docs

### PR DESCRIPTION
**Pull request by CrepeGoat**
This PR moves a lot of the formatting changes in MideTechnology/endaq-python's documentation into the docs of this repo.

Docs rendered at https://mide-technology-idelib.readthedocs-hosted.com/en/feature-update-docs/

### CODE REVIEW

File: .readthedocs.yaml [hunk 1]
```diff
@@ -1,16 +1,29 @@
...
+    python: "3.8"
+
 # Build documentation in the docs/ directory with Sphinx
 sphinx:
-   configuration: docs/source/conf.py
+  configuration: docs/source/conf.py
+
+# Optionally build your docs in additional formats such as PDF
+formats:
+  - pdf
```
Comment (StokesMIDE) [inline .readthedocs.yaml:20]: Does this now render to PDF? Do we want this?

Comment (CrepeGoat) [reply .readthedocs.yaml:20]: I remember only that it was in an example I was using as a basis for writing the config file. But I think you're right in that it's not *necessary*.

Having said that, I just went to the docs and clicked the PDF button that I never saw until you pointed it out... and tbh it's kinda cool 🤷‍♂️

**Review by CFlaniganMide — APPROVED** [review]

File: .readthedocs.yaml (hunk 1 above)
Comment (CrepeGoat) [reply .readthedocs.yaml:20]: @StokesMIDE talking w/ Pete atm & trying to close up PR's. Can I consider this resolved?

**Review by StokesMIDE — APPROVED** [review]

File: .readthedocs.yaml (hunk 1 above)
Comment (StokesMIDE) [reply .readthedocs.yaml:20]: Go for it.

**Pull request merged**